src/pages/all-about-carbon/why-use-carbon copy.mdx

```diff
@@ -0,0 +1,259 @@
+---
+title: Why use Carbon?
```

Comment:
The page name has `copy` at the end of it. 

I'm also not seeing this page in the nav - is that intentional?

Reply:
Oh, I think I included one page by mistake. Let me check. Thank you!